### PR TITLE
fix: revise words on docs intro page

### PR DESCRIPTION
**Pull request by srindom**

### CODE REVIEW

File: www/apps/book/app/page.mdx [hunk 1]
```diff
@@ -44,13 +45,11 @@ npx create-medusa-app@rc
...
-Medusa is for ambitious businesses and developers that are limited by traditional ecommerce solutions:
+Medusa is for businesses that have unique requirements that other digital commerce platforms can't easily support. Businesses of any size can use Medusa, from small startups to large enterprises - all it takes is a developer who can run a few commands and deploy your project.
 
-- You want rich commerce features that are customizable and flexible enough to support your business use case as if you built them from scratch.
-- You want your custom requirements built as an integral part of the commerce application’s features and architecture, rather than hacked into it.
-- You want to customize your commerce flows by integrating external services without data inconsistency and irregularity.
+Below are some stories from companies that use Medusa:
 
 <CardList items={[
   {
```
Comment (srindom) [inline www/apps/book/app/page.mdx]: would really like these to use logos of the companies to make them feel more user story-ish.

File: www/apps/book/app/page.mdx [hunk 2]
```diff
@@ -8,14 +8,15 @@ export const metadata = {
 
 # {pageNumber} Introduction
 
-Medusa is a digital commerce platform with a built-in framework for customizations and integrations.
+Medusa is a digital commerce platform with a built-in framework for customization.
 
-Medusa's framework includes two types of tools:
+Medusa is ships with three main tools:
```
Comment (shahednasser) [inline www/apps/book/app/page.mdx]:
```suggestion
Medusa ships with three main tools:
```

File: www/apps/book/app/page.mdx [hunk 3]
```diff
@@ -8,14 +8,15 @@ export const metadata = {
...
 
-1. Modules that provide commerce functionalities, such as Product, Order, and Cart modules.
-2. Customization tools to build your business features, such as custom data models, business logic, flows, and endpoints.
+1. A suite of commerce modules that provide core commerce functionalities, such as tracking inventory, calculating cart totals, accepting payments, managing orders, and much more.
+2. A framework to build custom functionality that is specific to your business, product, or industry. This includes tools for introducing custom API endpoints, business logic, and data models; building workflows and automations; and integrating with third-party services.
```
Comment (shahednasser) [inline www/apps/book/app/page.mdx]:
```suggestion
2. A framework to build custom functionalities specific to your business, product, or industry. This includes tools for introducing custom API endpoints, business logic, and data models; building workflows and automations; and integrating with third-party services.
```

File: www/apps/book/app/page.mdx [hunk 4]
```diff
@@ -8,14 +8,15 @@ export const metadata = {
...
+3. A customizable admin dashboard for merchants to configure and operate their store.
 
-With these tools, you save time you would spend with other platforms on maintaining and setting up your customizations. Instead, you rapidly build important features for your business.
+When you start Medusa the first time you get a fully fledged commerce platform with all the features you need to get off the ground. But unlike other platforms, Medusa is built with customization in mind. This avoids the need to build hacky workarounds that are difficult to maintain and scale, and instead allows you to focus on building features that move the needle.
```
Comment (shahednasser) [inline www/apps/book/app/page.mdx]:
```suggestion
When you start Medusa the first time you get a fully fledged commerce platform with all the features you need to get off the ground.

But, unlike other platforms, Medusa is built with customization in mind. You don't need to build hacky workarounds that are difficult to maintain and scale. Instead, you can focus on building features that move the needle.
```

Not a fan of " you can focus on building features that move the needle." as it feels vague. What do you think of something like " you can focus on building features that brings your business's vision to life"?

File: www/apps/book/app/page.mdx [hunk 5]
```diff
@@ -44,13 +45,11 @@ npx create-medusa-app@rc
 
 ---
 
-## Who is Medusa for
+## Who should use Medusa
 
-Medusa is for ambitious businesses and developers that are limited by traditional ecommerce solutions:
+Medusa is for businesses that have unique requirements that other digital commerce platforms can't easily support. Businesses of any size can use Medusa, from small startups to large enterprises - all it takes is a developer who can run a few commands and deploy your project.
```
Comment (shahednasser) [inline www/apps/book/app/page.mdx]:
```suggestion
Medusa is for businesses with unique requirements that other digital commerce platforms can't easily support. Businesses of any size can use Medusa, from small startups to large enterprises - all it takes is a developer to manage and deploy your project.
```

"a developer who can run a few commands" feels somewhat demeaning 😅

File: www/apps/book/app/page.mdx (hunk 1 above)
Comment (shahednasser) [reply www/apps/book/app/page.mdx]: I can handle this in a separate PR 👍🏻

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by srindom** [comment]
@shahednasser - can I ask you to take over on this PR. Feel free to adjust wording as you see fit

**Review by shahednasser — APPROVED** [review]

**Pull request merged**